nilearn/plotting/html_stat_map.py

```diff
@@ -49,11 +50,12 @@ def _data_to_sprite(data):
 
 def _threshold_data(data, threshold=None):
     """ Threshold a data array.
-        Returns: data (masked array), threshold (updated)
+        Returns: data (array), mask (boolean array) threshold (updated)
     """
     # If threshold is None, do nothing
     if threshold is None:
-        return data, threshold
+        mask = np.full(data.shape, True)
```

Comment:
So actually this mask is never used in the case of threshold=None. @GaelVaroquaux suggested we could just return `None`, but I tried to do something the same shape for consistency. I had used True as that will have all values as meeting the "condition", see [comment](https://github.com/nilearn/nilearn/pull/1957/files#r275800077)

Reply:
> So actually this mask is never used in the case of threshold=None. @GaelVaroquaux suggested we could just return `None`, but I tried to do something the same shape for consistency

it's true that it's never used anywhere, but I prefer your solution which returns something of the same type and shape, it's less likely to cause hard-to-find bugs. still, since the mask is True for thresholded values below, it should probably be filled with False here